memoization/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (C) 2012-2021, TomTom (http://tomtom.com).
+  ~ Copyright (C) 2012-2022, TomTom (http://tomtom.com).
```

Comment:
To avoid having to touch every file yearly, legal removed the end year requirement. We also adopted this in our repos. E.g., https://github.com/tomtom-internal/tomtom-android-tools/blob/master/.idea/copyright/TomTom.xml.

It can now just be the date the file/project was created.